packages/mobile/src/components/details-tile/DetailsProgressInfo.tsx

```diff
@@ -0,0 +1,113 @@
+import type { SearchTrack, Track } from '@audius/common'
+import {
+  formatLineupTileDuration,
+  playbackPositionSelectors
+} from '@audius/common'
+import { View } from 'react-native'
+import { useSelector } from 'react-redux'
+
+import IconCheck from 'app/assets/images/iconCheck.svg'
+import Text from 'app/components/text'
+import { makeStyles } from 'app/styles'
+import { useThemeColors } from 'app/utils/theme'
+
+import { ProgressBar } from '../progress-bar'
+
+const { getTrackPosition } = playbackPositionSelectors
+
+const messages = {
+  played: 'Played',
+  unplayed: 'Unplayed',
+  inProgress: 'In Progress',
+  timeLeft: 'left'
+}
+
+const useStyles = makeStyles(({ palette, spacing, typography }) => ({
+  progressInfoSection: {
+    gap: spacing(2),
+    width: '100%',
+    marginBottom: spacing(4),
+    paddingHorizontal: spacing(4)
+  },
+
+  progressInfoContainer: {
+    flexDirection: 'row',
+    justifyContent: 'space-between'
+  },
+
+  progressTextContainer: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    alignContent: 'center',
+    gap: 2
+  },
+
+  progressInfoText: {
+    fontSize: typography.body1.fontSize,
+    color: palette.neutralLight4
+  },
+
+  progressTextSlider: {
+    height: 4,
+    width: '100%',
+    marginTop: 0,
+    marginBottom: 0
+  }
+}))
+
+type DetailsProgressInfoProps = {
+  track: Track | SearchTrack
+}
+
+export const DetailsProgressInfo = ({ track }: DetailsProgressInfoProps) => {
+  const { duration } = track
+  const { neutralLight4 } = useThemeColors()
+  const styles = useStyles()
+  const playbackPositionInfo = useSelector((state) =>
+    getTrackPosition(state, { trackId: track.track_id })
+  )
+
+  const progressText = playbackPositionInfo
+    ? playbackPositionInfo.status === 'IN_PROGRESS'
+      ? messages.inProgress
+      : messages.played
+    : messages.unplayed
+
+  const durationText =
+    playbackPositionInfo?.status === 'IN_PROGRESS'
+      ? `${formatLineupTileDuration(
+          duration - playbackPositionInfo.playbackPosition,
+          true
+        )} ${messages.timeLeft}`
+      : formatLineupTileDuration(duration, true)
+
+  return (
+    <View style={styles.progressInfoSection}>
+      <View style={styles.progressInfoContainer}>
+        <Text style={styles.progressInfoText} weight='demiBold'>
+          {durationText}
+        </Text>
+        <View style={styles.progressTextContainer}>
+          <Text style={styles.progressInfoText} weight='demiBold'>
+            {progressText}
+          </Text>
+          {playbackPositionInfo?.status === 'COMPLETED' ? (
+            <IconCheck
+              style={{ marginBottom: 4 }}
+              height={16}
```

Comment:
you can actually import "spacing" utility directly to do `spacing(4)` if ya want